### PR TITLE
Fix Typos

### PR DESCRIPTION
**Pull request by loveboylion**

### CODE REVIEW

File: model/mod_list.py [hunk 1]
```diff
@@ -319,7 +319,7 @@ def eventFilter(self, source_object: QObject, event: QEvent) -> None:
                         # Open Steam URI with Steam action
                         elif (
                             action == open_mod_steam_action
-                        ):  # ACTION: Open steam:// uri in Steam
+                        ):  # ACTION: Open steam:// url in Steam
```
Comment (twstagg) [inline model/mod_list.py:322]: This is not a typo...

https://en.wikipedia.org/wiki/Uniform_Resource_Identifier

File: view/main_content_panel.py [hunk 2]
```diff
@@ -1693,7 +1693,7 @@ def _do_save_animation(self) -> None:
         logger.debug("Active mods list updated")
         if (
             self.active_mods_panel.list_updated  # This will only evaluate True if this is initialization, or _do_refresh()
-            and not self.actions_panel.save_button_flashing_animation.isActive()  # No need to reenable if it's already blinking
+            and not self.actions_panel.save_button_flashing_animation.isActive()  # No need to re enable if it's already blinking
```
Comment (twstagg) [inline view/main_content_panel.py:1696]: Technically, this should be "re-enable" :stuck_out_tongue:

File: view/main_content_panel.py [hunk 3]
```diff
@@ -2490,7 +2490,7 @@ def _do_build_database_thread(self) -> None:
 
     def _do_edit_steam_webapi_key(self) -> None:
         """
-        Opens a QDialogInput that allows the user to edit their Steam apikey
+        Opens a QDialogInput that allows the user to edit their Steam api key
```
Comment (twstagg) [inline view/main_content_panel.py:2493]: Technically, this should be "API-key" :stuck_out_tongue:

File: window/settings_panel.py [hunk 4]
```diff
@@ -440,7 +440,7 @@ def buildDatabaseBtnContextMenuEvent(self, point: QPoint) -> None:
         )  # db builder expiry
         set_steam_apikey = contextMenu.addAction(
             "Set Steam WebAPI key"
-        )  # steam webapi key
+        )  # steam web api key
```
Comment (twstagg) [inline window/settings_panel.py:443]: Technically, this should be "API-key" stuck_out_tongue

**Review by twstagg — CHANGES_REQUESTED** [review]

**Review by twstagg — APPROVED** [review]

File: view/main_content_panel.py (hunk 3 above)
Comment (loveboylion) [reply view/main_content_panel.py]: done

File: window/settings_panel.py (hunk 4 above)
Comment (loveboylion) [reply window/settings_panel.py]: done

**Pull request merged**